documentation/docs/instructor/lecture-series.mdx

```diff
@@ -0,0 +1,34 @@
+---
+id: lecture-series
+title: Lecture Series
+sidebar_label: Lecture Series
+---
+
+import lectureSeriesInputVisualization from './assets/lecture-series/LectureSeriesInputVisualization.png';
+import createLectureButtonImage from './assets/lecture-series/CreateLectureButton.png';
+import lectureCreationModeSelectImage from './assets/lecture-series/LectureCreationModeSelect.png';
+import lectureSeriesImage from './assets/lecture-series/LectureSeriesCreation.png';
+import Image, {ImageSize} from "../../src/components/Image/Image";
+import lectureCardsImage from './assets/lecture-series/LectureCards.png';
+import lectureCardRemoveButtonImage from './assets/lecture-series/LectureCardRemoveButton.png';
+import lectureCardEditButtonImage from './assets/lecture-series/LectureCardEditButton.png';
+
+# Lecture Series
+Lecture series allow you to create multiple lectures without specifying their contents. You can use this to scaffold your lectures at the beginning of the semester and inform students when each lecture will take place.
+
+To open the lecture series creation page, navigate to the Lectures tab in the course management section, click <img src={createLectureButtonImage} alt={"Lecture Create Button"} style={{ verticalAlign: 'bottom', maxHeight: '1.5rem'}} />
+and select 'Lecture Series' in <img src={lectureCreationModeSelectImage} alt={"Lecture Creation Mode Select"} style={{ verticalAlign: 'bottom', maxHeight: '1.5rem'}} />.
```

Comment:
In the whole section, can we write `Lecture Series` in italic or highlight it with backticks? It would make it easier to read and establish the term better